src/wp-admin/includes/class-core-upgrader.php

```diff
@@ -333,34 +309,80 @@ public static function should_update_to_version( $offered_ver ) {
 				return false;
 
 			// Don't claim we can update on update-core.php if we have a non-critical failure logged.
-			if ( $wp_version == $failure_data['current'] && false !== strpos( $offered_ver, '.1.next.minor' ) )
+			if ( $cp_version == $failure_data['current'] && false !== strpos( $offered_ver, '.1.next.minor' ) )
```

Comment:
@invisnet do you know what this `.1.next.minor` text is?  Where does this come from, and do we need to do something else instead?

Reply:
I have no idea so I left it as-is. It's something to track down for testing.

Reply:
Found it - it is precisely what it says in the comment `// Don't claim we can update on update-core.php if we have a non-critical failure logged.`

The rest of the related code is here:

https://github.com/ClassicPress/ClassicPress/blob/8797d5eaf7a06de6814fd91d2d6c25fd8ff27bdb/src/wp-admin/update-core.php#L188-L199